scripts/eks_existing_vpc_locals.snip

```diff
@@ -1,5 +1,4 @@
   vpc_id                 = "{{ .VPCID }}"
-  private_route_table_id = "{{ .PrivateRouteTableID }}"
```

Comment:
Why did this change?

Reply:
This field is only needed to setup peering. On the UI, we were considering the HCP AWS -> EKS -> Existing VPC flow to create public 1.14.0 clusters by default and not setup peering. Hence, removed this.